docs/marionette.application.md

```diff
@@ -130,11 +130,11 @@ var App = Mn.Application.extend({
     console.log('Initialize');
   },
 
-  onBeforeStart: function(options) {
-    this.model = new MyModel(options.data);
+  onBeforeStart: function(app, options) {
+    this.model = new MyModel(this.options.data);
```

Comment:
Hi @scott-w  this.options is correct because the options are supplied in the constructor in the example http://marionettejs.com/docs/v3.0.0/marionette.application.html#application-lifecycle. Both app.options and this.options would be correct for this example. I could do the following to illustrate the difference between app.options (this.options) and options:

onBeforeStart: function (app, options) {
   if (options === undefined) {
       this.model = new MyModel(app.options.data); //same as this.options.data
   }
   else {
      this.model = new MyModel(options.data);
  }